### DISCUSSION TITLE
p/88862964f94e4bdca5c97a5a2581898a/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#17] [Announcements]
# p/88862964f94e4bdca5c97a5a2581898a/

场景最近换了 m2 macbook，使用大概一周多了，作为一个 windows 长期使用者，截止目前为止遇到了很多问题和疑惑，有些解决了，有些目前仍未找到解决方法。 如何添加字体之前日常的字体为更纱黑体，它并不出色，但主要优势是以统一的风格支持简繁日英四种语言。基本步骤和 windows 类似，下载 ttc 点击打开，等待验证通过即可。不过 mac 上这个过程会很慢很慢很慢以至于让我以为出现了 b

https://blog.rxliuli.com/p/88862964f94e4bdca5c97a5a2581898a/#%E5%A6%82%E4%BD%95%E6%9D%80%E6%AD%BB%E7%89%B9%E5%AE%9A%E8%BF%9B%E7%A8%8B

**Comment by Simon-is-a-slowcoach** [comment]
关于 [如何杀死特定进程](https://blog.rxliuli.com/p/88862964f94e4bdca5c97a5a2581898a/#%E5%A6%82%E4%BD%95%E6%9D%80%E6%AD%BB%E7%89%B9%E5%AE%9A%E8%BF%9B%E7%A8%8B) 的问题，其实可以在 活动监视器 中先选中想要杀死的进程，然后点击导航栏的 ⊗ 就可以了。

**Reply by rxliuli** [reply]
感谢，吾辈没留意到这个按钮

**Comment by huwan** [comment]
"如何在 Finder 当前目录打开终端" 这一条可以试试 [Ji4n1ng/OpenInTerminal](https://github.com/Ji4n1ng/OpenInTerminal)，推荐里面的Lite版本，[OpenInTerminal-Lite 和 OpenInEditor-Lite](https://github.com/Ji4n1ng/OpenInTerminal/blob/master/Resources/README-Lite-zh.md)，直接一键在当前目录打开终端、文本编辑器。

**Reply by rxliuli** [reply]
感谢，吾辈最终找到了自定义右键菜单的应用 SzContext，它更通用可以新增任意程序到右键菜单中，ref: https://github.com/RoadToDream/SzContext/

**Comment by wibus-wee** [comment]
关于鼠标反向的事情，其实可以在这里设置哦～

![](https://github-production-user-asset-6210df.s3.amazonaws.com/62133302/252404309-ab5d6dfc-5ccf-4841-b8d8-39e4230b02ff.png)

**Reply by rxliuli** [reply]
系统这个会影响触摸板，吾辈只想修改滚轮的

**Reply by wibus-wee** [reply]
唔，那好吧

**Comment by AlynxZhou** [comment]
关于禁止本地创建 `.DS_Store`，会有什么副作用吗？

**Reply by rxliuli** [reply]
没有发现副作用

**Reply by AlynxZhou** [reply]
我觉得最痛苦的不是这个 `.DS_Store`，是有时候目录里会出现很多 `._` + 正常文件的文件名的文件，不知道是什么，也不好排除

**Reply by rxliuli** [reply]
> 我觉得最痛苦的不是这个 `.DS_Store`，是有时候目录里会出现很多 `._` + 正常文件的文件名的文件，不知道是什么，也不好排除

例如呢？

**Reply by AlynxZhou** [reply]
就比如我有一个文件叫 `1.jpg`，拿到别的系统上看就会多一个 `._1.jpg`，我暂时没搞清楚这个文件是怎么来的……

**Comment by miranquil** [comment]
[如何使用类似 ctrl+home/ctrl+end 的快捷键跳转光标](https://blog.rxliuli.com/p/88862964f94e4bdca5c97a5a2581898a/#如何使用类似-ctrl-home-ctrl-end-的快捷键跳转光标)
实际上 macOS 自带全局的类 Emacs 快捷键（手动艾特@AlynxZhou）。^(Ctrl)+A=Home, ^+E=End,^+F=→, ^+B=←, ^+D=⌫。而如果把 Ctrl 换成 ⌥(option) 会以单词为单位执行对应操作。

**Reply by miranquil** [reply]
哦还有^+P=↑, ^+N=↓。
